### ISSUE TITLE
libgfortran=3.0 should not be install with numpy <= 1.9

### ISSUE THREAD

**Issue by jreback** [conda/conda#2177]
``libgfortran=1.0`` works just fine, but since ``libgfortran=3.0`` just came out, its not compatible with older numpies (using 1.8.1 here), see [here](https://travis-ci.org/pydata/pandas/jobs/113480881).

So this totally breaks on ubuntu.
```
jreback@ubuntu:~/pandas3.4$ conda create -n test python=3.4 numpy=1.8.1 libgfortran --dry
Using Anaconda Cloud api site https://api.anaconda.org
Fetching package metadata: ......
Solving package specifications: ............
Package plan for installation in environment /home/jreback/miniconda/envs/test:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    numpy-1.8.1                |           py34_0         7.2 MB

The following NEW packages will be INSTALLED:

    libgfortran: 3.0-0        
    numpy:       1.8.1-py34_0 
    openssl:     1.0.2g-0     
    pip:         8.0.3-py34_0 
    python:      3.4.4-0      
    readline:    6.2-2        
    setuptools:  20.1.1-py34_0
    sqlite:      3.9.2-0      
    tk:          8.5.18-0     
    wheel:       0.29.0-py34_0
    xz:          5.0.5-1      
    zlib:        1.2.8-0      

Dry run: exiting
jreback@ubuntu:~/pandas3.4$ 
```

cc @ilanschnell @msarahan @mcg1969

**Comment by zym1010** [comment]
same problem. tried numpy 1.9.3 without MKL and failed just upon importing.

**Comment by zym1010** [comment]
currently, I guess a simple solution is to specify libgfortran=1 in conda install.

seems to me recently these kinds of problems appear pretty often... can Continuum Analytics be more strict on QA? I'd like to pay some fee for quality bundles, instead of compiling everything myself...

**Comment by ilanschnell** [comment]
numpy `1.8.1` did not explicitly list `libgfortran` as a dependency, but `1.8.2` does:
```
$ conda create -n test python=3.4 numpy=1.8.2
```
works fine for me.

**Comment by jreback** [comment]
thanks @zym1010 updated to be <= 1.9 (we had a CI run that specifically tested with 1.8.1)

**Comment by jreback** [comment]
ahh, ok @ilanschnell thanks.

**Comment by ilanschnell** [comment]
Better, just say:
```
$ conda create -n test python=3.4 numpy=1.8
```

**Comment by jreback** [comment]
yeah, I had this hard-coded in the CI, so just left it. Looks ok now that I fixed the ``libgfortran=1.0``

thanks @ilanschnell

**Comment by zym1010** [comment]
@ilanschnell  seems that it's still broken for 1.9?

**Comment by mcg1969** [comment]
Unless there is a reason to believe this is the solver's fault---and not faulty metadata---this isn't something I can fix. Leave it to the metadata folks...

**Comment by insertinterestingnamehere** [comment]
It looks like we're missing a build of scipy 0.16.1 with the latest bugfix release of numpy 1.9. That's causing an older build of  See https://github.com/scipy/scipy/issues/5917#issuecomment-192548240.

The 1.9 failure here is specific to the nomkl package. Getting a build of that one with the newer libgfortran should fix it.

**Comment by ilanschnell** [comment]
I've updated `numpy 1.9` as well as `scipy 0.17.0 np19` to link against `libgfortran 3.0`.

**Comment by tritemio** [comment]
This (or a similar) issue is causing RTD to fail in importing scipy when using conda envs: https://github.com/rtfd/readthedocs.org/issues/2074

The error is triggered when importing `scipy.stats`:

```
ImportError: libgfortran.so.1: cannot open shared object file: No such file or directory
```
See the RTD issue for full logs.

**Comment by dprada** [comment]
@tritemio In my case this was solved updating conda to 4.0. After that numpy and scypi were also updated to:

```
    numexpr:      2.4.4-np110py27_p0  [mkl] --> 2.5-np110py27_0   
    numpy:        1.10.2-py27_p0      [mkl] --> 1.10.4-py27_1     
    pytables:     3.2.2-np110py27_0   --> 3.2.2-np110py27_1 
    scikit-learn: 0.17-np110py27_p1   [mkl] --> 0.17.1-np110py27_0
    scipy:        0.16.1-np110py27_p0 [mkl] --> 0.17.0-np110py27_2
```
